sample-app/src/main/java/com/sinch/sample/Utils.java

```diff
@@ -21,6 +22,9 @@ public class Utils {
   private static final String SMS_SERVICE_PLAN_ID = "SMS_SERVICE_PLAN_ID";
   private static final String SMS_SERVICE_PLAN_TOKEN = "SMS_SERVICE_PLAN_TOKEN";
 
+  private static final String MAILGUN_API_KEY = "MAILGUN_SERVICE_API_KEY";
```

Comment:
Nit: Why would you name the properties differently?

Reply:
From mailgun's dashboard, it is called "API key"
So, I  just mimic the term from dedicated dashboard
![image](https://github.com/user-attachments/assets/1fc9451f-c69d-45da-88c7-742313ae3f5a)